src/main/kotlin/com/dansplugins/fiefs/fief/FiefRepository.kt

```diff
@@ -0,0 +1,78 @@
+package com.dansplugins.fiefs.fief
+
+import com.dansplugins.fiefs.Fiefs
+import java.util.*
+
+/**
+ * Stores all Fiefs in a list and provides methods for accessing and modifying the list.
+ */
+class FiefRepository(private val plugin: Fiefs) {
+    private val fiefs = mutableListOf<Fief>()
```

Comment:
I assume eventually this will be backed by a DB, negating the need for the list in the repository, but this is fine for testing purposes.